### DISCUSSION TITLE
Root login for nginx-pm?

### DISCUSSION THREAD

**Discussion by OddSquirrel** [tteck/Proxmox#818] [Help Needed]
Yeah, so, try not to laugh. I just installed nginx-pm in a nice, comfy container. During the setup process, it asked me for a root password and I happily obliged.

Funnily enough, for the life of me, I can't manage to log in as root—neither via the Proxmox console nor through SSH.
IIRC, it didn't ask me for a specific login name, either, so I assume it should be root.

Granted, I'm not the sharpest tool in the shed, but would anybody have an idea what I might have done wrong? 🤔

**Comment by tteck** [comment]
Sounds like you forgot/misspelled you your password!

**Reply by OddSquirrel** [reply]
That's what would come to mind, I know. But luckily, the password isn't asterisked, so I was able to see and check before I hit enter.

The proper login name is _root_, though, right?

**Reply by tteck** [reply]
yes, `root`

**Reply by tteck** [reply]
This is for the lxc, not NPM

**Reply by OddSquirrel** [reply]
Yessir, I'm def talking about console access to the LXC. If I leave the password blank, it'll auto-login, right? Would that be considered risky for an outside-facing proxy? If not, I'll just run the whole show again without a root password.

**Reply by tteck** [reply]
Yes, during creation if left blank it will auto login that's my preferred method

**Reply by OddSquirrel** [reply]
Doesn't that open up a bunch of security issues? Or am I safe as long as I keep root via SSH off?
